Formula/kertish-dfs.rb

```diff
@@ -0,0 +1,30 @@
+class KertishDfs < Formula
+  desc "Kertish FileSystem and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dfs"
+  url "https://github.com/freakmaxi/kertish-dfs/archive/v21.2.0066.tar.gz"
+  sha256 "1ff300a803a0d3bd068e1492319a771912cccd4ba3a72098b6d05678813af966"
+  license "GPL-3.0-only"
+  head "https://github.com/freakmaxi/kertish-dfs.git"
+
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "mod", "download"
+    Dir.chdir("fs-tool") { system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtfs" }
+    Dir.chdir("admin-tool") { system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtadm" }
```

Comment:
```suggestion
    cd "fs-tool" { system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtfs" }
    cd "admin-tool" { system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtadm" }
```
I don't remember if `cd` accepts blocks in braces -- please check this locally first. Otherwise, you can do `cd "fs-tool" do ... end`, etc, which is closer to what other formulae do.

Reply:
I'll change it as you suggested